webrisk/src/test/java/webrisk/SnippetsIT.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2020 Google LLC
```

Comment:
Is this new code?  Then it should be 2022

Reply:
Not new code. I just renamed the file and made some updates.